Modules/_hacl/Hacl_Streaming_HMAC.c

```diff
@@ -2519,4 +2521,3 @@ Hacl_Streaming_HMAC_agile_state
     "unreachable (pattern matches are exhaustive in F*)");
   KRML_HOST_EXIT(255U);
 }
```

Comment:
```suggestion
}

```
Please revert unrelated changes (line below, I cannot provide a suggestion on the line).